v-spec.adoc

```diff
@@ -4118,6 +4118,7 @@ possibly binary logical functions of the two input masks:
 | 0 | 0 | 0 | 1 | vmand.mm vd, src1, src2     |
 | 1 | 0 | 0 | 1 | vmxnor.mm vd, src1, src2    |
 | 0 | 1 | 0 | 1 | vmand.mm vd, src2, src2     | vmmv.m vd, src2
+| 0 | 1 | 1 | 1 | vmor.mm vd, src1, src2      |
 | 1 | 1 | 0 | 1 | vmorn.mm vd, src2, src1     |
 | 0 | 0 | 1 | 1 | vmand.mm vd, src1, src1     | vmmv.m vd, src1
 | 1 | 0 | 1 | 1 | vmorn.mm vd, src1, src2     |
```

Comment:
wouldn't it be better added here, after 
| 1 | 0 | 1 | 1 | vmorn.mm vd, src1, src2 
?
following the binary sequence.

Reply:
Yeah, I didn't realize the table was sorted.  I'll fix it.

Reply:
3ec63c12ae2bd4c68c3540afa6ab38ff442ed148